### PR TITLE
Added support for virtual products

### PR DESCRIPTION
**Pull request by lfluvisotto**

### CODE REVIEW

**Comment by lfluvisotto** [comment]
Does anymore look at the pull requests?

**Comment by rikterbeek** [comment]
Hi @lfluvisotto,

Thanks for the PR! We are looking into it.

Regards,
Rik
Adyen

**Comment by rikterbeek** [comment]
Hi @lfluvisotto,

could you look at the comment in the PR review? if this is fixed we can merge it.

Thanks,
Rik
Adyen

**Comment by lfluvisotto** [comment]
File upgrade-1.0.2-1.0.3.php  renamed to upgrade-1.3.1-1.3.2.php

File config.xml edited to <version>1.3.2</version>

File: app/code/community/Adyen/Subscription/sql/adyen_subscription_setup/upgrade-1.0.2-1.0.3.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?php
```
[the remark this answers is not in the document]
Comment (lfluvisotto) [reply app/code/community/Adyen/Subscription/sql/adyen_subscription_setup/upgrade-1.0.2-1.0.3.php]: File upgrade-1.0.2-1.0.3.php renamed to upgrade-1.3.1-1.3.2.php

File config.xml edited to 1.3.2

**Comment by lfluvisotto** [comment]
@rikterbeek could you review the PR again, thanks.

**Comment by rikterbeek** [comment]
@lfluvisotto,

Thanks for changing this!. I will merge it into the code.

Regards,
Rik
Adyen

**Pull request merged**